packages/@tailwindcss-upgrade/src/codemods/template/migrate-modernize-arbitrary-values.ts

```diff
@@ -146,6 +147,83 @@ export function migrateModernizeArbitraryValues(
           continue
         }
 
+        // Migrate `@media` variants
+        //
+        // E.g.: `[@media(scripting:none)]:` -> `noscript:`
+        if (
+          // Only top-level, so `in-[@media(scripting:none)]` is not supported
```

Comment:
`in-[@media(scripting:none)]` is not valid in the first place so maybe a different example would be better here?